### DISCUSSION TITLE
apiflask中能不能提供全局响应模板？

### DISCUSSION THREAD

**Discussion by wangzhatian** [apiflask/apiflask#155] [Q&A] [answered]
例如
`{
	"code": "",
	"data": {
		"bug_id": "",
	},
	"msg": "",
	"size": 0,
	"success": true
}`

**Comment by greyli** [comment, marked as answer]
参考这里：https://apiflask.com/schema/#base-response-schema-customization

这里有一个示例：https://github.com/greyli/apiflask/blob/main/examples/base_response/app.py
